connectors/protocol/connectors.py

```diff
@@ -793,6 +793,12 @@ async def prepare(self, config, sources):
         if is_main_connector and self.features.features != source_klass.features():
             doc["features"] = source_klass.features()
             self.log_debug("Populated features")
+        elif (
+            self.native
+            and self.features.features.keys() != source_klass.features().keys()
+        ):
+            doc["features"] = source_klass.features() | self.features.features
+            self.log_debug("Added features missing from Native definitions")
```

Comment:
```suggestion
        elif (
            self.native
            and self.features.features.keys() != source_klass.features().keys()
        ):
            missing_features = source_klass.features().keys() - self.features.features.keys()
            doc["features"] = source_klass.features() | self.features.features
            self.log_debug(f"Added missing features [{', '.join(missing_features)}] from Native definitions")
```

IMO it could be useful to also log, which features were missing, so it's easier to update on Kibana side, if someone wants to update the features.